packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+import shieldIcon from './images/shield.svg';
+
+import { Form, Input, Checkbox, OnValidateFunction } from '../fields';
+import {
+  State as ValidatorState,
+  useValidatorState,
+  MiddlewareReducer as ValidatorMiddlewareReducer,
+} from '../../lib/validator';
+
+import './index.scss';
+
+export type NewUserEmailFormProps = {
+  getString?: (id: string) => string;
+  validatorInitialState?: ValidatorState;
+  validatorMiddlewareReducer?: ValidatorMiddlewareReducer;
+};
+
+export const NewUserEmailForm = ({
+  getString,
+  validatorInitialState,
+  validatorMiddlewareReducer,
+}: NewUserEmailFormProps) => {
+  const validator = useValidatorState({
+    initialState: validatorInitialState,
+    middleware: validatorMiddlewareReducer,
+  });
+
+  return (
+    <Form
+      data-testid="new-user-email-form"
+      validator={validator}
+      className="new-user-email-form"
+    >
+      <Localized id="new-user-sign-in-link">
+        <p className="sign-in-copy" data-testid="sign-in-copy">
+          Already have a Firefox account? <a>Sign in</a>
+        </p>
+      </Localized>
+      <Localized id="new-user-email" attrs={{ placeholder: true, label: true }}>
+        <Input
+          type="text"
+          name="new-user-email"
+          label="Enter your email"
+          data-testid="new-user-email"
+          placeholder="foxy@mozilla.com"
+          required
+          spellCheck={false}
+          onValidate={(value, focused, props) => {
+            return validateEmail(value, focused, props, getString);
+          }}
+        />
+      </Localized>
+
+      <Localized id="new-user-confirm-email" attrs={{ label: true }}>
+        <Input
+          type="text"
+          name="new-user-confirm-email"
+          label="Confirm your email"
+          data-testid="new-user-confirm-email"
+          required
+          spellCheck={false}
+          onValidate={(value, focused, props) =>
+            validateEmail(value, focused, props, getString)
```

Comment:
Since we need to ensure the value of second email input is identical to the value of the first email input, we cannot have the same validation function for both inputs.

Reply:
Thanks for pointing this out, it probably simplifies things to only match the two for the second input